### PR TITLE
MEN-4534: Add UpdateControlMapExpirationTimeSeconds to mender.conf

### PR DESCRIPTION
**Pull request by oleorhagen**
See: https://tracker.mender.io/browse/MEN-4534 for info

### CODE REVIEW

File: conf/config.go [hunk 1]
```diff
@@ -90,7 +90,8 @@ type MenderConfig struct {
 }
 
 type DBusConfig struct {
-	Enabled bool
+	Enabled                               bool
+	UpdateControlMapExpirationTimeSeconds int
```
Comment (kacf) [inline conf/config.go:94]: It should be outside the `DBusConfig` structure. It is not DBus specific, DBus is just one of the ways that it can be updated.

**Review by kacf — CHANGES_REQUESTED** [review]
One structural change is needed, but looks good otherwise.

**Comment by oleorhagen** [comment]
@mender-test-bot start pipeline

File: conf/config.go [hunk 2]
```diff
@@ -200,6 +202,13 @@ func loadConfigFile(configFile string, config *MenderConfig, filesLoadedCount *i
 		return err
 	}
 
+	if config.MenderConfigFromFile.UpdateControlMapExpirationTimeSeconds == 0 {
+		log.Info("'DBus.UpdateControlMapExpirationTimeSeconds' is not set " +
```
Comment (kacf) [inline conf/config.go:206]: ~DBus.~

**Review by kacf — APPROVED** [review]
One wrong log message below, looks good otherwise!

**Comment by oleorhagen** [comment]
@mender-test-bot start pipeline :pray:

**Pull request merged**